crates/types/src/config_loader.rs

```diff
@@ -134,19 +134,10 @@ impl ConfigLoader {
                     .map(|_| "rocksdb-total-memory-size".into()),
             );
 
-        let fig = match Env::var("DO_NOT_TRACK").as_deref() {
+        match Env::var("DO_NOT_TRACK").as_deref() {
             Some("yes" | "1" | "true") => fig.join(("disable-telemetry", true)),
             Some("no" | "0" | "false") => fig.join(("disable-telemetry", false)),
             _ => fig,
-        };
-
-        if let Some(no_proxy) = Env::var("NO_PROXY") {
-            fig.join((
-                "no-proxy",
-                no_proxy.split(',').map(str::trim).collect::<Vec<_>>(),
-            ))
```

Comment:
How can users override the no_proxy config option via the env variable `NO_PROXY`? Is this automatically read by any of the hyper utils that we are using?

Reply:
read a little higher up :D. turns out we were setting it twice, the second time doing the comma parsing, i guess because I didn't want to parse strings like '["a", "b"]'

Reply:
Ah, makes sense. Thanks for the clarifcation.